nautobot/extras/models/jobs.py

```diff
@@ -1116,13 +1127,13 @@ def create_schedule(
         """
 
         if interval == JobExecutionType.TYPE_IMMEDIATELY:
-            start_time = timezone.now()
+            start_time = timezone.localtime()
             name = name or f"{job_model.name} - {start_time}"
         elif interval == JobExecutionType.TYPE_CUSTOM:
             if start_time is None:
                 # "start_time" is checked against models.ScheduledJob.earliest_possible_time()
                 # which returns timezone.now() + timedelta(seconds=15)
-                start_time = timezone.now() + timedelta(seconds=20)
+                start_time = timezone.localtime() + timedelta(seconds=20)
```

Comment:
`.now()` always returns UTC, whereas `.localtime()` uses the TIME_ZONE setting.

https://docs.djangoproject.com/en/4.2/ref/utils/#module-django.utils.timezone

Since we use `start_time.tzinfo` to set the `ScheduledJob.time_zone`, this is an important fix for consistent behavior when using a non-default TIME_ZONE.